### ISSUE TITLE
Generalizar para que cada vez que se ejecute genere un archivo distinto.

### ISSUE THREAD

**Issue by JonAnderDiez** [JonAnderDiez/ExpediaScraper#9]
La intención es que se ejecute una vez al mes, por lo que cada vez que se ejecute deberá generar un CSV con un nombre diferente tal y como se especifica en el documento Word.

**Comment by benataranburu** [comment]
¿No lo hace ya?

**Comment by JonAnderDiez** [comment]
Cierto, lo he escrito mal, lo que quería poner es que cada vez que se ejecute genere 12 archivos nuevos referentes a los siguientes 12 meses que es el periodo en el que los datos pueden tener validez. Si me deja cambio el título.
